modules/apps/site/site-cms-site-initializer/test/js/main/util/convertArrayToFormattedString.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {convertArrayToFormattedString} from '../../../../src/main/resources/META-INF/resources/js/main/util/convertArrayToFormattedString';
+
+describe('convertArrayToFormattedString', () => {
+	it('return an empty string for an empty array', () => {
+		expect(convertArrayToFormattedString([])).toBe('');
+	});
+
+	it('return the string itself for a single-element array', () => {
+		expect(convertArrayToFormattedString(['First Item'])).toBe(
```

Comment:
This is not a real problem, but usually we prefer to use `toEqual` instead of `toBe`. As we're comparing strings here, it is fine, but if it was an object, it would fail even if the values are equal.
https://dev.to/thejaredwilcurt/why-you-should-never-use-tobe-in-jest-48ca